uhabits-core/src/main/java/org/isoron/uhabits/core/ui/widgets/WidgetBehavior.java

```diff
@@ -61,6 +61,12 @@ public void onRemoveRepetition(@NonNull Habit habit, Timestamp timestamp)
         performToggle(habit, timestamp);
     }
 
+    public void onCancelRepetition(@NonNull Habit habit, Timestamp timestamp)
```

Comment:
Instead of creating a new method that does essentially the same thing as `onRemoveRepetition`, I think it's fine if we just make `onRemoveRepetition` call `notificationTray.cancel(habit)`.